dev/test-wheel.sh

```diff
@@ -21,4 +21,11 @@ if [ -z "$(python -m pyroma ./ | grep 'Final rating: 10/10')" ]; then
 fi
 echo "Pyroma wheel check: done"
 
+echo "Check-wheel-content wheel check: start"
+check-wheel-contents ./dist/ --ignore W002
+# The ignore W002 is "Wheel contains duplicate files"
+# It is required because of the autogenerated proto files
+# There is no file-based ignore for this tool
```

Comment:
```suggestion
# The ignore W002 is "Wheel contains duplicate files"
# It is required because of the autogenerated proto files
# There is no file-based ignore for this tool
check-wheel-contents ./dist/ --ignore W002
```